Formula/cargo-release.rb

```diff
@@ -16,16 +16,50 @@ class CargoRelease < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "d63c11ca314240c51f31f66765f8daa1dcfd8ba9a7feb01de8eff9ec21096da1"
   end
 
-  depends_on "rust" # uses `cargo` at runtime
+  depends_on "pkg-config" => :build
+  depends_on "rust" => :build
+  depends_on "rustup-init" => :test
+  depends_on "libgit2"
```

Comment:
```suggestion
  depends_on "libgit2@1.5"
```

Reply:
I added the following from `cargo-c`. Shall we wait and see what happens first?

```ruby
ENV["LIBGIT2_SYS_USE_PKG_CONFIG"] = "1"
ENV["LIBSSH2_SYS_USE_PKG_CONFIG"] = "1"
```

Reply:
Yes, you're right! This has worked on #134068. I'll reflect these changes here right away.

Reply:
Yes, you're right! This has worked on #134068. I'll reflect these changes here right away.

Reply:
Done.